examples/c/CMakeLists.txt

```diff
@@ -54,13 +54,23 @@ function(prepare_executable executable)
 endfunction()
 
 if(PHI3)
-  add_executable(phi3 ${CMAKE_SOURCE_DIR}/src/phi3.cpp)
-  prepare_executable(phi3)
+  if(USE_CXX)
+    add_executable(phi3 ${CMAKE_SOURCE_DIR}/src/cpp/phi3.cpp)
+    prepare_executable(phi3)
+  else()
+    add_executable(phi3 ${CMAKE_SOURCE_DIR}/src/c/phi3.cpp)
+    prepare_executable(phi3)
+  endif()
```

Comment:
You can set the SOURCE_PATH variable to `${CMAKE_SOURCE_DIR}/src/cpp/` when `USE_CXX` is true, else `${CMAKE_SOURCE_DIR}/src/c` and avoid needing to do this when preparing the executable.